scipy/stats/_stats_py.py

```diff
@@ -310,6 +310,12 @@ def hmean(a, axis=0, dtype=None):
         dtype of `a`, unless `a` has an integer `dtype` with a precision less
         than that of the default platform integer. In that case, the default
         platform integer is used.
+    weights : array_like, optional
+        The weights array can either be 1-D (in which case its length must be
+        the size of `a` along the given `axis`) or of the same shape as `a`.
+        Default is None, which gives each value a weight of 1.0.
+
+    .. versionadded:: 1.9
```

Comment:
Missing indentation should be under the parameter
```suggestion
        .. versionadded:: 1.9
```